middleman-core/features/support/env.rb

```diff
@@ -4,6 +4,9 @@
 require 'simplecov'
 SimpleCov.root(File.expand_path(File.dirname(__FILE__) + '/../..'))
 
+require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
```

Comment:
From "this line" on PhantomJS is required to be installed on the build system, if a feature is flagged with `@javascript`